### PR TITLE
[BD-2880] Content Cards now base re-eligibilty on send time

### PR DESCRIPTION
**Pull request by internetisaiah**
### Summary

We've updated Content Cards to base re-eligibility strictly on "send time", and not based on impression time. This does not apply to the new "at first impression" cards.

### Related PRs, issues, or features (optional)

- [BD-2880](https://jira.braze.com/browse/BD-2880)

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Comment by internetisaiah** [comment]
Hey @davidbielik , I'm new to this feature, so let me know if I'm understanding your request correctly. Thanks!

**Comment by froodian** [comment]
is the comment from @diana-smirnova on BD-2880 still relevant/correct? https://jira.braze.com/browse/BD-2880?focusedId=219203&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-219203

> we haven't yet updated content card re-eligibility to strictly use impression time (ideally this will be implemented in the future). Docs should clarify that we use the greater of send time or last impression time to determine re-eligibility

**Comment by internetisaiah** [comment]
@froodian , I'm not sure! Do you know, @diana-smirnova ? If not, do you know the best place for me to ask? Thanks!

**Comment by exigoking** [comment]
yeah I think what happened is that we wanted it to be strictly based on "send time" but where we left it was 

> In the current state, re-eligibility for non-JIT content cards is based on the later of send time and last impression time.


outlined in this doc https://docs.google.com/document/d/1qdoEhx10pYikw1nI_iMW8uZ8MVX6y_da-wqG3tocHsg/edit

**Comment by internetisaiah** [comment]
Ah i see, so what is the current state of the feature @exigoking ? Is the information in my PR currently incorrect?

**Comment by exigoking** [comment]
> Ah i see, so what is the current state of the feature @exigoking ? Is the information in my PR currently incorrect?

I think the part where we mention `all other Content Card types use send time` we should change to `all other Content Card types use the latest of send time and impression time`

File: _docs/_user_guide/message_building_by_channel/content_cards/create.md [hunk 1]
```diff
@@ -209,14 +209,18 @@ You can specify multiple custom events and purchases that should remove a card f
 
 ### Updating launched cards
 
-Content Cards cannot be edited after they are sent. If you find you need to make changes to cards that have already been sent, consider the following approaches.
+Content Cards cannot be edited after they are sent. If you find you need to make changes to cards that have already been sent, consider using [Campaign Re-eligibility]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/reeligibility/) as shown in the following options.
+
+{% alert note %}
+Content Cards using [at first impression]({{site.baseurl}}/user_guide/message_building_by_channel/content_cards/create/card_creation/#differences-between-creating-cards-at-launch-or-entry-versus-at-first-impression) use impression time to calculate re-eligibility&#8212;however, all other Content Card types use send time.
```
Comment (internetisaiah) [inline _docs/_user_guide/message_building_by_channel/content_cards/create.md:215]:
```suggestion
Content Cards using [at first impression]({{site.baseurl}}/user_guide/message_building_by_channel/content_cards/create/card_creation/#differences-between-creating-cards-at-launch-or-entry-versus-at-first-impression) use impression time to calculate re-eligibility&#8212;however, all other Content Card types use whichever send time or impression time is latest.
```

Comment (internetisaiah) [reply _docs/_user_guide/message_building_by_channel/content_cards/create.md:215]: @exigoking , hows this?

Comment (exigoking) [reply _docs/_user_guide/message_building_by_channel/content_cards/create.md:215]: lgtm

**Review by mark-at-braze — APPROVED** [review]
Happy w/ the latest update. 👍👍

**Pull request merged**